articles/rules-are-dangerous.md

```diff
@@ -0,0 +1,46 @@
+---
+authorName: Rolf Bruderer
+authorGithubUsername: bruderol
+issue: 31
+title: Rules are Dangerous
+---
+# {{page.title}}
+
+Rules followed blindly in a dogmatic way do a lot of harm. This is an uncomfortable truth especially for us as engineers. As such we are used to see the world in rigid rules, true and false, if and else. We like to have crystal clear and exact definitions, guidelines or best practices on how to do something. This is not bad in the first place. But we all must be very careful that we do not fall into the trap of just following such rules blindly without thinking well about consequences. 
```

Comment:
Thanks for feedback,I dont want to make it longer and do not have time anymore to optimize this. I think a lot of what you are sayng should be implicitly clear out of my intro (written between the lines). 

And: this is not just about collaboration rules!! Not at all! This is about all rules! Often we as technician think that what I say here does not count for very technical rules because technical rules are different, more precise and generally true. but I thnk this is wrong. The same problem as well applies to technical rules. Which is why there are also some technical examplesin the text.